.spi.yml

```diff
@@ -1,5 +1,4 @@
 version: 1
 builder:
   configs:
-    - swift_version: 5.9
```

Comment:
Should we update the version to `5.10` (I'll update the version with my Package.swift version at the same time later)?

If we should remove this field, then what scenario is this field for? I could not find other meaningful scenario of the field then.